mern-ecommerce/server.js

```diff
@@ -34,7 +34,7 @@ const connectDB = async () => {
     process.exit(1);
   }
 };
-connectDB();
+connectdb();
```

Comment:
The function call `connectdb();` should be `connectDB();` to match the function definition `const connectDB = async () => { ... }`. The function name is case-sensitive, and JavaScript will throw a ReferenceError if it cannot find a function with the exact name.